src/robot-arm-element.ts

```diff
@@ -0,0 +1,106 @@
+import { html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ElementPin } from '.';
+
+@customElement('wokwi-robot-arm')
+export class RobotArmElement extends LitElement {
+  /**
+   * Angle of shoulder
+   */
+  @property() sholderAngle = 297;
+
+  /**
+   * Angle of elbow
+   */
+  @property() elbowAngle = 52;
+
+  /**
+   * Angle of wrist
+   */
+  @property() wristAngle = 56;
+
+  /**
+   * Angle of gripper
+   */
+  @property() gripperAngle = 270;
+
+  /**
+   * The color of the gripper
+   */
+  @property() gripperColor = '#eeeedd';
+
+  /**
+   * The color of the arm
+   */
+  @property() armColor = '#ffffdd';
+
+  /**
+   * The color of the joints
+   */
+  @property() jointColor = '#111111';
+
+  readonly pinInfo: ElementPin[] = [
```

Comment:
We also need GND/VCC pins (they are not currently used in the simulation, but we do want the schematic to look as close as possible to reality)

Reply:
ok, will add the two pins. I was cheating since I knew they had no function.